### ISSUE TITLE
minikube 1.12.3 fails to interpret memory unit correctly

### ISSUE THREAD

**Issue by rkialashaki** [kubernetes/minikube#9057]
**Steps to reproduce the issue:** 

1.  Mac OSX Catalina 10.15.5
2. Docker Desktop for Mac 2.3.0.4
3. brew install minikube 
4. From minikube help: 
```
--memory='': Amount of RAM to allocate to Kubernetes (format: <number>[<unit>], where unit = b, k, m or g).
```
5. minikube start --cpus 6 --memory 16g

Notice I specify the memory allocation for the minikube container should be 16GB, however, in the output below, you will see minikube creates a docker container of memory 8100MB instead. 

**Full output of failed command:** 
```
😄  minikube v1.12.3 on Darwin 10.15.5
✨  Automatically selected the docker driver. Other choices: hyperkit, virtualbox
👍  Starting control plane node minikube in cluster minikube
🔥  Creating docker container (CPUs=6, Memory=8100MB) ..
```

**Full output of `minikube start` command used, if not already included:**

**Optional: Full output of `minikube logs` command:**
<details>

</details>

**Comment by unixlab** [comment]
I think this is a duplicate of #9017 and already fixed in PR #9033

So I would assume that we just need to wait until the fix get released in a 1.12.4 bug fix release or wait for 1.13.0

**Comment by unixlab** [comment]
I think this is a duplicate of #9017 and already fixed in PR #9033.

So I would assume that we just need to wait until the fix get released in a 1.12.4 bug fix release or wait for 1.13.0

**Comment by sharifelgamal** [comment]
Correct, this will be fixed with our 1.13.0 release which should be out soon. Sorry for the bad experience here. I'll close this as a dupe.
